manual/ssl/README.md

```diff
@@ -149,7 +152,8 @@ Netty-tcnative provides the native integration with OpenSSL. Follow
 [these instructions](http://netty.io/wiki/forked-tomcat-native.html) to
 add it to your dependencies.
 
-Note that using netty-tcnative requires JDK 1.7 or above.
+Note that using netty-tcnative requires JDK 1.7 or above and requires the
+presence of OpenSSL on the system.  It will not fall back to the JDK implementation.
```

Comment:
maybe clarify the last sentence: "It will _not_ fall back to the default JDK implementation, therefore you should only use it if you are sure that OpenSSL is installed on your system."